modules/olm-changing-update-channel.adoc

```diff
@@ -5,40 +5,24 @@
 [id="olm-changing-update-channel_{context}"]
 = Changing the update channel for an Operator
 
-The Subscription of an installed Operator specifies an update channel, which is
-used to track and receive updates for the Operator. To upgrade the Operator to
-start tracking and receiving updates from a newer channel, you can change the
-update channel in the Subscription.
+The Subscription of an installed Operator specifies an update channel, which is used to track and receive updates for the Operator. To upgrade the Operator to start tracking and receiving updates from a newer channel, you can change the update channel in the Subscription.
 
-The names of update channels in a Subscription can differ between Operators, but
-should follow a common convention within a given Operator. Some naming
-schemes include:
-
-- `4.3`, `4,4`, `4.5`
-- `stable-4.3`, `stable-4.4`, `stable-4.5`
-- `alpha`, `beta`, `stable`, `latest`
-
-Alternatively, the channel names might follow the version numbers of the
-application provided by the Operator.
+The names of update channels in a Subscription can differ between Operators, but the naming scheme should follow a common convention within a given Operator. For example, channel names might follow a particular stream of updates for a minor release of the application provided by the Operator (`1.2`, `1.3`) or a simple release frequency (`stable`, `fast`).
```

Comment:
Actual content change here. Basically removing references to OCP version-based naming schemes.

Reply:
No, that's good, I was worried about how long this sentence got.